pkgs/tools/audio/mpdris2/default.nix

```diff
@@ -16,7 +16,7 @@ stdenv.mkDerivation rec {
   '';
 
   buildInputs = [ intltool autoreconfHook pythonPackages.wrapPython ];
-  propagatedBuildInputs = with pythonPackages; [ python pygtk dbus-python ];
+  propagatedBuildInputs = with pythonPackages; [ python pygtk dbus-python mutagen ];
```

Comment:
this only works, when you are in a `nix-shell`.
Otherwise you will see the following:

```
$ /nix/store/123j3zdik295fjnbgcrlsrf5zmgx97vk-mpDris2/bin/mpDris2
...
2017-08-11 23:40:27,583 .mpDris2-wrapped INFO: Mutagen not available, covers in music files will be ignored.
...
```